docs/documentation/github-desktop.md

```diff
@@ -0,0 +1,40 @@
+# Store your code online with GitHub and GitHub Desktop
```

Comment:
I was just looking at the titles listed under 'Basic usage' in the [Tutorials and page templates](https://govuk-prototype-kit-pr-994.herokuapp.com/docs/tutorials-and-examples).

All of the other titles start with gerunds ('Sending,' 'Adding' and so on). To be consistent, should we change 'Store' to 'Storing'?

And a more general question: in the titles, is there a reason some begin with simple present tense and others begin with gerunds?

Reply:
That's a good rationale! Present sense is more emphatic alright, so I can see why we'd want to use it in the context of a tutorial.

The [GOV.UK style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) doesn't say anything about either approach being better. I think we'll be fine as long as all titles in a given section follow the same approach where possible.

Apart from for consistency's sake, would 'storing' also make sense here because not all users of this section will be using this GitHub Desktop doc? (Since some users will be using the doc on setting up Git, right?)

Reply:
sounds good, lets gerund it up